connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerSinkTaskThreadedTest.java

```diff
@@ -561,6 +588,51 @@ private void expectStopTask(final long expectedMessages) throws Exception {
         return EasyMock.expectLastCall();
     }
 
+    @SuppressWarnings("unchecked")
+    private IExpectationSetters<Object> expectRebalanceDuringPoll() throws Exception {
```

Comment:
Always a toss up with test code like this whether to try to make these helper methods with more options or just clearer but a bit repetitive by using completely different methods. This seems fine for now, we might want to rethink/refactor if we end up with another one or two methods similar to this.